src/statmodels.jl

```diff
@@ -154,6 +154,9 @@ predict(obj::RegressionModel) = error("predict is not defined for $(typeof(obj))
 predict!(obj::RegressionModel) = error("predict! is not defined for $(typeof(obj)).")
 dof_residual(obj::RegressionModel) = error("dof_residual is not defined for $(typeof(obj)).")
 
+## Tests for statistical models
+function testfit() # Pairwise sequential tests of model fit
```

Comment:
The syntax to declare a function without any method is `function testfit end`. Again, no need to add a comment "Tests for statistical models" since there's only one function under this heading at this point.